infrastructure/food_oasis_app_stack/README.md

```diff
@@ -0,0 +1,62 @@
+# ballotnav infrastructure configuration
+
+These files describe the load balancer and webserver infrastructure of the ballotnav app using [`terraform`](https://www.terraform.io/) HCL. 
+
+Additional configuration and environment-specific values are stored in a private
+github repo.
+
+# Getting started
+To apply this configuration in a new environment:
+
+## prerequisites
+- `terraform` v0.12.0+
+- aws account and credentials created 
+- `awscli` installed and configured [see docs here](https://aws.amazon.com/cli/)
+- all required secrets are created & stored in SSM, see below:
+
+### secrets stored in SSM
+This configuration should create the initial deployment and infra for a load
```

Comment:
Please update language for food oasis project

Reply:
I'll make all the changes in the README soon